tools/backups2datalad.py

```diff
@@ -113,7 +113,7 @@ def main(
     logging.basicConfig(
         format="%(asctime)s [%(levelname)-8s] %(name)s %(message)s",
         datefmt="%Y-%m-%dT%H:%M:%S%z",
-        level=logging.INFO,
+        level=logging.DEBUG,
```

Comment:
let's just add use of e.g `BACKUP2DATALAD_LOG_LEVEL` env var which would provide numeric value to pass (or do it more properly with cli option etc), so we do not need to change the script for this and I will just merge and I will try it out to identify.  Will be useful in the long term anyways

Reply:
`--log-level` option added (Requires `click-loglevel` package).

Reply:
ha -- you made an entire package for that? heheh ;)
ok to out of draft so I could merge?

Reply:
Taken out of draft.